sys/linux/test/fuse_deadlock

```diff
@@ -4,8 +4,8 @@
 mkdirat(0xffffffffffffff9c, &AUTO='./file0\x00', 0x0)
 r0 = openat$fuse(0xffffffffffffff9c, &AUTO='/dev/fuse\x00', 0x2, 0x0)
 mount$fuse(0x0, &AUTO='./file0\x00', &AUTO='fuse\x00', 0x0, &AUTO={{'fd', 0x3d, r0}, 0x2c, {'rootmode', 0x3d, 0x4000}, 0x2c, {'user_id', 0x3d, 0x0}, 0x2c, {'group_id', 0x3d, 0x0}, 0x2c, {[], [], 0x0}})
-read$FUSE(r0, &AUTO=""/4096, AUTO)
+read$FUSE(r0, &AUTO={AUTO, 0x0, <r1=>0x0, 0x0, 0x0, 0x0, 0x0, ""/8192}, AUTO)
```

Comment:
I wonder how it was working before with 4096... do you have any hypothesis?

Reply:
I haven't found any documentation on that, but looking at the [source code](https://github.com/torvalds/linux/blob/master/fs/fuse/dev.c#L1199) it seems that if the total read buffer length is less than FUSE_MIN_READ_BUFFER, a read on /dev/fuse returns EINVAL.

Was it working without returning any error (I haven't tested this specific script, only a C program interacting with /dev/fuse directly)?